apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -200,6 +200,99 @@ module.exports = {
+              fields: [
+                {
+                  label: "Title",
+                  name: "title",
+                  widget: "string",
+                  required: false,
+                },
+                {
+                  label: "Description",
+                  name: "description",
+                  widget: "string",
+                  required: false,
+                },
+                {
+                  label: "Open Graph",
+                  name: "openGraph",
+                  widget: "object",
+                  fields: [
+                    {
+                      label: "Title",
+                      name: "title",
+                      widget: "string",
+                      required: false,
+                    },
+                    {
+                      label: "Description",
+                      name: "description",
+                      widget: "string",
+                      required: false,
+                    },
+                    {
+                      label: "Images",
+                      name: "images",
+                      widget: "list",
+                      required: false,
+                      fields: [
+                        {
+                          name: "url",
+                          label: "URL",
+                          widget: "string",
+                        },
+                        {
+                          name: "width",
+                          label: "Width",
+                          widget: "string",
+                        },
+                        {
+                          name: "height",
+                          label: "Height",
+                          widget: "string",
+                        },
+                        {
+                          name: "alt",
+                          label: "Alt Text",
+                          widget: "string",
+                        },
+                      ],
+                    },
+                  ],
+                },
+                {
+                  label: "Twitter",
+                  name: "twitter",
+                  widget: "object",
+                  fields: [
+                    {
+                      label: "Handle",
+                      name: "handle",
+                      widget: "string",
+                      required: false,
+                      default: "@Code4Africa",
+                    },
+                    {
+                      label: "Site",
+                      name: "site",
+                      widget: "string",
+                      required: false,
+                      default: "@Code4Africa",
+                    },
+                    {
+                      label: "Card Type",
+                      name: "cardType",
+                      widget: "hidden",
+                      default: "summary_large_image",
+                    },
+                  ],
+                },
+              ],
+            },
```

Comment:
Lets update the structure to 👇🏽 . I don' think we need some of these in every page e.g. cardType, @site, etc., should be the same across the site.
```suggestion
            {
              label: "SEO",
              name: "seo",
              widget: "object",
              fields: [
                {
                  label: "Title",
                  name: "title",
                  widget: "string",
                },
                {
                  label: "Description",
                  name: "description",
                  widget: "text",
                  required: false,
                  pattern: ["^.{1,150}$", "Must be up to 156 characters"],
                },
                {
                  label: "Meta data",
                  name: "meta",
                  widget: "object",
                  hint: "Search engines support",
                  fields: [
                    {
                      label: "Title",
                      name: "title",
                      widget: "string",
                      required: false,
                      hint: "Uses page title if not provided",
                      pattern: ["^.{1,70}$", "Must be up to 70 characters"],
                    },
                    {
                      label: "Description",
                      name: "description",
                      widget: "text",
                      required: false,
                      hint: "Uses page description if not provided",
                      pattern: ["^.{1,150}$", "Must be up to 156 characters"],
                    },
                  ],
                },
                {
                  label: "Open Graph",
                  name: "open-graph",
                  hint: "Facebook, Slack, and other social media platforms",
                  widget: "object",
                  fields: [
                    {
                      label: "Title",
                      name: "title",
                      hint: "Uses page title if not provided",
                      widget: "string",
                      required: false,
                    },
                    {
                      label: "Description",
                      name: "description",
                      widget: "string",
                      hint: "Uses pae description if not provided",
                      required: false,
                    },
                    {
                      label: "Images",
                      name: "images",
                      widget: "list",
                      min: 1,
                      max: 1,
                      required: false,
                      fields: [
                        {
                          name: "url",
                          label: "URL",
                          widget: "string",
                        },
                        {
                          name: "width",
                          label: "Width",
                          widget: "number",
                        },
                        {
                          name: "height",
                          label: "Height",
                          widget: "number",
                        },
                        {
                          name: "alt",
                          label: "Alt Text",
                          widget: "string",
                        },
                      ],
                    },
                  ],
                },
                {
                  label: "Twitter",
                  name: "twitter",
                  widget: "object",
                  fields: [
                    {
                      label: "Handle",
                      name: "handle",
                      hint: "@username of content creator",
                      widget: "string",
                      required: false,
                      default: "@Code4Africa",
                    },
                  ],
                },
              ],
            },
```